src/components/EducationCard.tsx

```diff
@@ -76,17 +81,16 @@ export function CompactYield({ quantity, isQuantityExact }: EducationGraphicProp
     <CompactGraphic
       title={t`Recipient`}
       icon={<Icon name="vault-recipient" fontSize="large" />}
-      subtext={`${t`Receives yield from`} ${isQuantityExact ? "" : "≈ "}${quantity} sOHM`}
+      subtext={`${t`Receives Yield from`} ${isQuantityExact ? "" : "≈ "}${quantity} sOHM`}
     />
   );
 }
 
-export function ArrowGraphic() {
-  // We hard-code 25px so that the arrows are vertically center-aligned with other graphics
+export function ArrowGraphic({ fill = "#999999", marginTop = "25px" }: ArrowGraphicProps) {
```

Comment:
same color here

Reply:
This default was typically overridden, but I've removed the default and found a better way of achieving this using theme-defined colours.